ssl/statem/statem_lib.c

```diff
@@ -1697,23 +1726,68 @@ int ssl_choose_client_version(SSL *s, int version)
     for (vent = table; vent->version != 0; ++vent) {
         const SSL_METHOD *method;
         int err;
+#ifndef OPENSSL_NO_TLS13DOWNGRADE
+        static const unsigned char tls11downgrade[] = {
+            0x44, 0x4f, 0x57, 0x4e, 0x47, 0x52, 0x44, 0x00
+        };
+        static const unsigned char tls12downgrade[] = {
+            0x44, 0x4f, 0x57, 0x4e, 0x47, 0x52, 0x44, 0x01
```

Comment:
Could do. I actually had that at one point in development. The problem I had was you have to declare them with an explicit size if you put the declarations in a header - otherwise sizeof doesn't work. I thought the implicit size was better, and no big deal to have them declared twice. I don't have a strong opinion either way though, so I'll go with whatever the consensus is.